src/ingest_llm_as/observability/langfuse_client.py

```
@@ -6,207 +6,120 @@
-        self.client.score(
-            trace_id=trace_id,
-            name=name,
-            value=value,
-            comment=comment,
-            metadata=metadata or {},
-        )
-    
+
+        try:
+            self.client.score_current_trace(name=name, value=value, comment=comment)
+        except Exception as e:
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)